lib/editor-config-error.ts

```diff
@@ -0,0 +1,22 @@
+class EditorConfigError extends Error {
+	fileName: string = '<anonymous>';
+	lineNumber: number = 1;
+	columnNumber: number = 1;
+	message: string = '';
+	rule: string = '';
+	source: string = '';
+	name: string = 'EditorConfigError';
```

Comment:
I think you can safely remove the types here in this section, as they will be inferred for you. Example:
```js
source = '';
name = 'EditorConfigError';
```